static/js/settings_display.js

```diff
@@ -5,25 +5,21 @@ var exports = {};
 exports.set_night_mode = function (bool) {
     var night_mode = bool;
     var data = { night_mode: JSON.stringify(night_mode) };
-    var context = {
-        enable_text: data.night_mode === "true" ?
-            i18n.t("enabled") :
-            i18n.t("disabled"),
-    };
```

Comment:
Shall I remove the context from other settings too? Since we are not using it anywhere now?